core/src/test/kotlin/org/evomaster/core/search/algorithms/StandardGeneticAlgorithmTest.kt

```diff
@@ -12,17 +12,20 @@ import org.evomaster.core.search.algorithms.onemax.OneMaxIndividual
 import org.evomaster.core.search.algorithms.onemax.OneMaxModule
 import org.evomaster.core.search.algorithms.onemax.OneMaxSampler
 import org.evomaster.core.search.service.ExecutionPhaseController
+import org.evomaster.core.search.service.Randomness
 import org.junit.jupiter.api.Test
 
 import org.junit.jupiter.api.Assertions.*
+import org.evomaster.core.search.algorithms.strategy.FixedSelectionStrategy
+import org.evomaster.core.search.algorithms.observer.GARecorder
 
 class StandardGeneticAlgorithmTest {
 
     val injector: Injector = LifecycleInjector.builder()
```

Comment:
if you are having more than 1 test in this class, this is problematic. shared injector creates dependencies. should be recreated for each test, eg, in a `@BeforeEach` method